### PR TITLE
Leave protocol of user submitted website urls untouched

### LINKED ISSUE #26: BUG: user website URL is forced to HTTPS protocol
if a user sets her website URL to: http://something.com
it will show up on her profile as: something.com
but will open: https://something.com

the protocol should be respected as provided by the user

### PR DESCRIPTION
**Pull request by greyvugrin**
Fixes #26

### CODE REVIEW

File: whydJS/app/controllers/LibUser.js [hunk 1]
```diff
@@ -469,8 +469,9 @@ function renderUserLinks(lnk){
 
 	// make sure URLs are valid
 	for (var i in lnk)
-		if (lnk[i])
+		if (lnk[i] && i !== "home") {
```
Comment (adrienjoly) [inline whydJS/app/controllers/LibUser.js]: Nice fix, but it opens the door to `javascript:` URLs => vulnerability...

Need to make sure that the user-provided URL is either http:// or https:// protocol.

**Comment by greyvugrin** [comment]
@adrienjoly thanks for the input, I've adjusted to account for that possibility. Tested with http, https, and any other protocol. If https is matched it used that, otherwise it falls back to http.

**Comment by adrienjoly** [comment]
good job, @greyvugrin !

**Comment by greyvugrin** [comment]
Thanks! 💪

**Pull request merged**